examples/volume_plane_rendering.py

```diff
@@ -4,18 +4,15 @@
 """
 import napari
 import numpy as np
-from qtpy.QtCore import Qt
-from qtpy.QtWidgets import QWidget, QGroupBox, QVBoxLayout, QHBoxLayout
+from napari.utils.translations import trans
 from skimage import data
-from superqt import QLabeledDoubleSlider
-
 
 viewer = napari.Viewer(ndisplay=3)
```

Comment:
```suggestion
viewer = napari.Viewer(ndisplay=3)
viewer.theme = "dark"
```
I am really looking forward to using this feature! :)
My default that I use napari is the light mode, so it took me a few minutes to figure out that this example needs dark mode to be visible. This suggestion should make it straightforward to avoid this issue.

Reply:
wait, is hard coding `dark` the right move here? is this a blending issue in `light` theme?
you're not the only one that uses `light` most of the time!

Reply:
I'm sure it's just that @alisterburt always uses dark and didn't notice the issue :P Maybe we can come up with some example data that works in both cases?

Reply:
You are right, I've just experimented a bit and e.g. changing line13 to:
```py
volume_layer = viewer.add_image(
    blobs, rendering='mip', name='volume',colormap="magenta", blending='opaque', opacity=0.25
)
```
would work in dark and light mode:
![image](https://user-images.githubusercontent.com/44469195/149653737-0b0aba19-e67f-482b-8045-247118a72a5d.png)

![image](https://user-images.githubusercontent.com/44469195/149653724-a7e27582-3988-482f-9fb0-947423691972.png)

I don't know why, but the very first frame displayed in the viewer in light mode is broken:
The opacity of the volume is very high, but as soon as I click once in the viewer, it changes to the desired opacity one can see in the above screenshot:
![image](https://user-images.githubusercontent.com/44469195/149653714-7a59d80b-92a3-4dff-a7ab-00c743a05a55.png)

Reply:
I think `blending='opaque'` implies opacity = 1. 
To control opacity you need `blending='translucent'`
And yea, I've tested this in my napari—which is Light mode—and the example as is gives a white square due to the additive blending issue in Light mode (see: https://github.com/napari/napari/issues/3374 & https://github.com/napari/napari/discussions/3914)
I assume many, if not most, examples could be bitten by this!

Reply:
thanks for your interest in the PR @kolibril13 ! As @psobolewskiPhD pointed out this is a blending issue with our white canvas and @brisvag is correct, this affects a few of our examples.

@kevinyamauchi and @psobolewskiPhD have been making great progress on this front so I'll leave this as-is for now and hope that the problem 'solves itself' soon 😆